orchagent/vnetorch.cpp

```diff
@@ -2913,6 +2918,11 @@ bool VNetRouteOrch::handleTunnel(const Request& request)
             mac = MacAddress(mac_list[idx_ip]);
         }
 
+        if (is_local)
```

Comment:
Just to clarify, this rule will only be added only if "check_directly_connected" is set in the VNET_RT_TUNNEL table correct?